### ISSUE TITLE
USER ISSUE: Claim Crash

### ISSUE THREAD

**Issue by LantzDryden** [StrangeLoopGames/EcoIssues#24870]
\
**User Id: (Not logged user)**
**Version:** 0.10.2.0 beta release-640

**Steps to Reproduce:**
1. Logged in private server after update
2. Claim was over allowed plots because claim papers weren't counting
3. opened claim stake and removed papers
4. replaced papers - got message about claim papers not counting until joining owner settlement (I don't have a settlement set up)
5. game crashes

I have recreated this several times.

**Expected behavior:**
Claim should have updated or given me information about how to fix the 'settlement' issue. 

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Components.PlotsComponent.DescribePlotsClaimed()
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
Outer Exceptions:
Exception has been thrown by the target of an invocation.
Exception has been thrown by the target of an invocation.</size>

```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Please specify what type of claim stake and claim papers you used.

**Comment by D3nnis3n** [comment]
Fixed in 10.2.1.
